client/client.go

```diff
@@ -365,7 +365,7 @@ func (c *authenticatingClient) createServiceURLs() error {
 				errorPrefix,
 				strings.Join(possibleRegions, ", "))
 		} else {
-			return errors.New(errorPrefix)
+			return fmt.Errorf(errorPrefix)
```

Comment:
just revert to errors.New() in this instance. Only need the Errorf() variant when there are formatting args to worry about.